bacon/controller/MainPageViewController.swift

```diff
@@ -11,6 +11,8 @@ import UIKit
 class MainPageViewController: UIViewController {
 
     var core: CoreLogic?
+    var currentMonthTransactions = [Transaction]()
+    var monthCounter = (0, 0)
```

Comment:
Whats a monthCounter? It seems to store the month and year?

Reply:
Yeah actually I am a bit unsure about this too. Like it is not really being used in the main page (it is used in the transaction list page to know which month to load) but because I want to minimise the number of times I load transactions, I had to load out the current month transactions in the main page (so that I can pass them into any other page), and to do that I need to calculate the current month/year. Since I calculated it already I thought I might as well save it so that I don't need to calculate again in the transaction list page.

Reply:
Oh i see i see, perhaps rename the variable? currentMonthYear? Cause monthCounter is very vague, like I don't what are you counting?